src/hssm/hssm.py

```diff
@@ -219,9 +217,17 @@ def __init__(
         link_settings: Literal["log_logit"] | None = None,
         prior_settings: Literal["safe"] | None = None,
         extra_namespace: dict[str, Any] | None = None,
+        missing_data: bool = False,
+        deadline: bool = False,
+        na_value: float = -999.0,
```

Comment:
maybe `missing_data_value` here instead of `na_value`?

Reply:
I am fine with both. `na_value` is shorter and is also a convention in `pandas`. Let me know which one you prefer

Reply:
If we have `missing_data` kwarg, then probably go with `missing_data_value` kwarg to make it consistent. 

We can also allow both `missing_data_value` and `na_value` and pool internally, if you think it's worth the burden. 

In general appreciate the convention argument, but locally I think going for `missing_data` AND `na_value` is confusing.